README.md

```diff
@@ -60,12 +60,19 @@ Alternatively, you can manually edit Application.xml and then reload the Wowza a
   </Property>
 ```
 
-### To deploy plugin code to VM:
-
-- (ensure Wowza application on VM has the properties above)
-- (update version file)
-- (tag version on github)
-- (trigger jenkins build that creates versioned deployment jar artifact)
+### To deploy a new version of the plugin code to the VM:
+
+- Ensure the Wowza application on the VM has the properties mentioned above.
+- Release a new version of the plugin:
+  - Update `conf/version`.  This should be just the version number itself, e.g. `1.7.1` or `2.0.0-beta`.
+  - Create a version tag and release notes:
+    - Tag the version in the git repo using the `v` prefix, e.g. `v1.7.1` or `v2.0.0-beta` (corresponding with the `conf/version` example).
+    - Create a Github release for the tag with brief notes about changes since the last release.
+  - Notes:
+    - It's probably easiest to do the tag and release creation in one shot in the Github UI.  If you choose to tag from the command line, don't forget to push tags to Github.
+    - If the head of the branch to be built isn't tagged to correspond to `conf/version`, the standard `deploymentJar` task will fail (and will suggest invoking `deploymentJarRelaxed`, which doesn't impose the same constraint).  E.g., if `version` contains `3.2.1`, the head of the branch to be built should be tagged as `v3.2.1`.
+- Trigger the Jenkins build that creates the versioned .jar artifact for deployment.  Assuming the code at the head of the branch is versioned/tagged properly, the plugin .jar should get built and copied to the artifacts directory, from which Puppet should be able to pick up the new version for deployment.  To start the build:
```

Comment:
Maybe mention the specific jenkins build here ("dlss-wowza-deploy" or whatever you called it).